ga4gh/client.py

```diff
@@ -543,10 +549,11 @@ def searchIndividuals(self, datasetId, name=None):
         request.dataset_id = datasetId
         request.name = pb.string(name)
         request.page_size = pb.int(self._pageSize)
-        return self._runSearchRequest(
+        return self._run_search_request(
             request, "individuals", protocol.SearchIndividualsResponse)
 
-    def searchReadGroupSets(self, datasetId, name=None, bioSampleId=None):
+    def search_read_group_sets(
+            self, datasetId, name=None, bioSampleId=None):
```

Comment:
So, for keyword parameters like these we do need to snake_case to ``bio_sample_id`` or it'll be very confusing. I think it's as well to just snake_case this entire module and be done with it.